### DISCUSSION TITLE
Rust 1.70, lazy_static and once_cell crates no longer needed ?

### DISCUSSION THREAD

**Discussion by lwandrebeck** [pest-parser/pest#863] [Ideas]
Hello,

With [Rust 1.70](https://blog.rust-lang.org/2023/06/01/Rust-1.70.0.html), we could get rid of lazy_static and once_cell crates as OnceCell and OnceLock are now part of the standard library.
But, that would imply to change MSRV.

What do you think of it ? Plan it for 2.7 ? 3.0 ? Never ? (or anything else in between ?).

**Comment by tomtau** [comment]
Yes, we could eventually bump MSRV, but we should revisit MSRV of popular crates that use pest: https://github.com/pest-parser/pest/issues/671#issuecomment-1199915940 and decide based on that.
